visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.html

```diff
@@ -1,3 +1,18 @@
-<md-button id="scrBtn" class="toolbar-button" ng-click="$ctrl.makeScreenshot()" title="ScreenShot ({{ $ctrl.SCREENSHOT_HOTKEY }})">
+<md-button
```

Comment:
In the title attribute of the button, I would show both hotkey options at the same time. The setting should control the default behavior on click, but not the available hotkeys: 
If Clipboard is enabled: `title="Copy screenshot to clipboard (CTRL+X), export it as a file by CTRL+Y"`
If Clipboard is not enable: `title="Export screenshot as file (CTRL+Y), copy it to clipboard by CTRL+X"`